scripts/generate-artifacts.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
should we have a 

```bash
set -e
```

here? I think we should probably add it to the rest of our bash scripts as well